Cargo.toml

```diff
@@ -8,14 +8,14 @@ rust-version = "1.61.0"
 
 [dependencies]
 anyhow = "1.0"
-c2pa = { version = "0.2", features = ["file_io"] }
+c2pa = { git = "https://github.com/contentauth/c2pa-rs.git", features = ["file_io"] }
```

Comment:
I'd like to see .github/workflows/publish.yml updated to fast-fail a PR if this is inadvertently left in place.

I'm not normally a fan of commenting things out, but I think it might be wise in this case so that we have a reminder of the required format when we do prepare to publish.

Reply:
We should do that. This has been updated to version = "0.3.0"